### PR TITLE
Revive and improve DELTA probe bounds, LRFB options

### PR DESCRIPTION
**Pull request by InsanityAutomation**
Followup - @Roxy-3D @marcio-ao  @thinkyhead 

Just the next step as discussed. Should let everyone see where im going at least.

Has the 3 point definition done as per Roxy's suggestion, delta forced to fixed points so its functional, and min_probe_edge definable per side. Has not seen a machine yet or even been compiled a bunch of ways. Travis will be done with it before I can get it on hardware tonight and im sure ill have more to do then.

### CODE REVIEW

**Comment by InsanityAutomation** [comment]
SO now this passes travis, brings in what @Roxy-3D wanted for 3 point, gets delta back to working.

@marcio-ao I cloned youre repo last night and poked over the Taz6 config. Ill go through and make sure i have everything suitable for it before this is all done. I just need to decide on using the config in head or youre customized branches, and if custom merge it with the relevant stuff from head and this first. Ironically those are the only machines I have not running my own configs, yours have been good enough that it wasnt worth maintaining for the few things id do differently!

After the day job... A bit more than I can do between other tasks!

**Comment by marcio-ao** [comment]
Could you add a warning about `[FRONT/BACK/RIGHT/LEFT]_PROBE_BED_POSITION` being deprecated? I can see a lot of our customers running into this issue.

**Comment by InsanityAutomation** [comment]
> 
> 
> Could you add a warning about `[FRONT/BACK/RIGHT/LEFT]_PROBE_BED_POSITION` being deprecated? I can see a lot of our customers running into this issue.

Absolutely. Ill also end up adding the pointer to the options in config adv. That was on tonights agenda.

**Comment by InsanityAutomation** [comment]
Delta and SCARA still use static values in places so those will need a followup at some point, but there should be no regressions at least. This should put to bed everything necessary immediately and properly document the config change.

**Comment by InsanityAutomation** [comment]
FYI, #15367 is necessary for travis to pass. Note the passed build here I had pulled it in, then pulled it back out again to avoid a conflict. With the two of these in travis should finally be happy again.

File: Marlin/Configuration_adv.h [hunk 1]
```diff
@@ -1337,6 +1337,42 @@
...
+ * Override if the automatically selected points are inadequate.
+ */
+#if EITHER(AUTO_BED_LEVELING_3POINT, AUTO_BED_LEVELING_UBL)
+  //#define PROBE_PT_1_X 15
+  //#define PROBE_PT_1_Y 180
+  //#define PROBE_PT_2_X 15
+  //#define PROBE_PT_2_Y 20
+  //#define PROBE_PT_3_X 170
+  //#define PROBE_PT_3_Y 20
+#endif
```
Comment (thinkyhead) [inline Marlin/Configuration_adv.h:1351]: Vera nice.

File: Marlin/src/lcd/extensible_ui/lib/lulzbot/language/language_en.h [hunk 2]
```diff
@@ -66,7 +66,7 @@ namespace Language_en {
   PROGMEM Language_Str AMAX_E3                  = u8"Amax E3";
   PROGMEM Language_Str AMAX_E4                  = u8"Amax E4";
   PROGMEM Language_Str JERK                     = u8"Jerk";
-  PROGMEM Language_Str JUNC_DEVIATION           = u8"Junc Dev";
+  PROGMEM Language_Str JUNCTION_DEVIATION       = u8"Junc Dev";
```
Comment (thinkyhead) [inline Marlin/src/lcd/extensible_ui/lib/lulzbot/language/language_en.h]: This causes bad things to happen. Revert! Revert!

**Comment by thinkyhead** [comment]
I'm terribly disappointed. There's nothing here to criticize! 😢

**Comment by InsanityAutomation** [comment]
This showed some of those junction deviation changes as I was bringing in the other PR and then reverting it just to make sure everything passed Travis. At least this brings everything to stable though there still a few more things I want to get done around this. I'll see how much time I have this weekend around the kids birthday party.

**Comment by tpruvot** [comment]
travis fix there: #15401

**Pull request merged**